api/views.py

```diff
@@ -35,19 +35,19 @@ class AssetViewSet(ModelViewSet):
 
     def get_queryset(self):
         user = self.request.user
+        try:
+            admin = self.request.user.is_staff
```

Comment:
Since this is a boolean, I think a more descriptive name would be `is_admin`.

Reply:
aah on it.